scripts/run_experiment.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+#SBATCH --partition=ephemeral
+#SBATCH --qos=ephemeral
+#SBATCH --ntasks=1
+#SBATCH --mem-per-cpu=8G
+#SBATCH --cpus-per-task=1
+#SBATCH --job-name=ADAM_EXPERIMENT
+#SBATCH --output=data/slurm_logs/%x-%j.out    # %x-%j means JOB_NAME-JOB_ID.
+#SBATCH --requeue
```

Comment:
TIL `--requeue` is an option you can specify instead of just manually queueing up a bunch of dependent jobs... 🙃

Reply:
I believe `--requeue` will just automatically requeue the job if its pre-empted by another job because of paritions. It's probably not needed for this script given the execution time is under an hour and ephemeral guarantees at least that long once a job starts.